fixtures/sku-init/sku-init.test.js

```diff
@@ -9,62 +9,84 @@ const projectName = 'new-project';
 const projectDirectory = path.join(fixtureDirectory, projectName);
 
 describe('sku init', () => {
-  beforeAll(async () => {
-    await fs.rm(projectDirectory, { recursive: true, force: true });
-  });
-
-  afterAll(async () => {
-    await fs.rm(projectDirectory, { recursive: true, force: true });
-
-    console.log(
-      "Running 'pnpm install' to clean up lockfile after sku-init test...",
-    );
-    await exec('pnpm install');
-    console.log('Cleanup complete');
-  });
+  let child;
+  let stdout;
+  let stderr;
 
-  it(
-    'should create a sku.config.ts',
+  beforeAll(
     async () => {
+      await fs.rm(projectDirectory, { recursive: true, force: true });
+
       const projectName = 'new-project';
       await fs.rm(path.join(fixtureDirectory, projectName), {
         recursive: true,
         force: true,
       });
 
-      const { child, stdout, stderr } = await runSkuScriptInDir(
+      ({ child, stdout, stderr } = await runSkuScriptInDir(
         'init',
         fixtureDirectory,
         [projectName],
-      );
+      ));
 
       console.log('sku init stdout');
       console.log(stdout);
       console.log('sku init stderr');
       console.error(stderr);
+    },
+    // `sku init` is a long running task and can take some time to complete
+    5 * 60 * 1000,
+  );
+
+  afterAll(async () => {
+    await fs.rm(projectDirectory, { recursive: true, force: true });
+    console.log(
+      "Running 'pnpm install' to clean up lockfile after sku-init test...",
+    );
+    await exec('pnpm install');
+    console.log('Cleanup complete');
+  });
 
-      expect(child.exitCode).toBe(0);
+  it('should exit with code 0', async () => {
+    expect(child.exitCode).toBe(0);
+  });
 
-      const skuConfig = await fs.readFile(
-        path.join(fixtureDirectory, projectName, 'sku.config.ts'),
-        'utf-8',
-      );
+  it('should create package.json', async () => {
+    const contents = await fs.readFile(
+      path.join(fixtureDirectory, projectName, 'package.json'),
+      'utf-8',
+    );
 
-      expect(skuConfig).toMatchInlineSnapshot(`
-"import type { SkuConfig } from 'sku';
+    expect(replaceDependencyVersions(JSON.parse(contents))).toMatchSnapshot();
```

Comment:
Whilst jest does support [Property Matchers](https://jestjs.io/docs/snapshot-testing#property-matchers), I chose to implement this by just replacing the version as it's easier to still test WHICH dependencies were installed (regardless of the version), and IMO it's a cleaner snapshot.